README.md

```diff
@@ -8,485 +8,18 @@
-
-```ts
-import { ngtx, asBool, toNativeElement } from '@centigrade/ngtx';
-
-const DIALOG_CANCEL = 'Cancel';
-
-describe(
-  'Some random tests as basic examples',
-  ngtx(
-    ({
-      useFixture,
-      detectChanges,
-      find,
-      findWhere,
-      triggerEvent,
-      attr,
-      textContent,
-    }) => {
-      let component: AnyComponent;
-      let fixture: ComponentFixture<AnyComponent>;
-
-      beforeEach(async(() => {
-        TestBed.configureTestingModule({
-          /* ... */
-        }).compileComponents();
-      }));
-
-      beforeEach(() => {
-        fixture = TestBed.createComponent(AnyComponent);
-        component = fixture.componentInstance;
-        useFixture(fixture);
-      });
-
-      it('[Wizard] should emit the finish-event when clicking on cancel button', () => {
-        // arrange
-        spyOn(component.finish, 'emit');
-
-        // act
-        const finishButton = findWhere(
-          (button) => textContent(button).includes(DIALOG_CANCEL),
-          ['.btn-footer-primary', '.btn-footer-secondary'],
-          toNativeElement,
-        );
-
-        finishButton.click();
-
-        // assert
-        const userAction: WizardResult = 'canceled';
-        expect(component.finish.emit).toHaveBeenCalledTimes(1);
-        expect(component.finish.emit).toHaveBeenCalledWith(userAction);
-      });
-
-      it('[TabsComponent] should set label translation to true', () => {
-        // arrange
-        const { componentInstance: wizard } = find(WizardComponent);
-
-        // act, assert
-        expect(wizard.translateTabs).toBe(true);
-      });
-
-      it('[Dialog] should emit closeDialog event on finish wizard', () => {
-        // arrange
-        spyOn(component.closeDialog, 'emit');
-
-        // act
-        triggerEvent('finishWizard', WizardComponent);
-
-        // asserrt
-        expect(component.closeDialog.emit).toHaveBeenCalledTimes(1);
-      });
-
-      it('[Textfield] should pass the readonly attribute', () => {
-        // arrange
-        const expectedValue = true;
-
-        // pre-condition
-        expect(attr('readOnly', 'input', asBool)).not.toBe(expectedValue);
-
-        // act
-        component.readonly = expectedValue;
-        detectChanges();
-
-        // assert
-        expect(attr('readOnly', 'input', asBool)).toBe(expectedValue);
-      });
-
-      it('[SomeView] should show a drop down with the currently selected item', () => {
-        // arrange
-        const expectedValue = 'selected item';
-        component.selectedItem = expectedValue;
-
-        // act
-        // passing "component" to "detectChanges"
-        // additionally runs ngOnInit and ngOnChanges
-        // on component, if they are defined:
-        detectChanges(component);
-
-        // assert
-        expect(textContent('.drop-down-label')).toEqual(expectedValue);
-      });
-    },
-  ),
-);
-```
-
-# Detailed Documentation
```

Comment:
could be moved to a separate markdown file in future, so that the README.md keep simple.

Reply:
This was already moved to the file `DOCUMENTATION.md` ;)